### DISCUSSION TITLE
Discord account disabled

### DISCUSSION THREAD

**Discussion by Yashgabani845** [sanjay-kv/Open-source-Practice#987] [Q&A] [answered]
please someone help me in this
i was just checking general discussion in discord and i just logged out
and my account got disabled i got main which says i have done one of the following violation 

-  Sending a large number of direct messages in a short span of time
- Participating in a server dedicated to spamming Discord
- Automating your user account or self-botting
- Taking other actions on Discord — for example, joining a ton of servers — faster than humanly possible
Attempting potentially fraudulent activity

but thing is have hardly done 20 messegs in Dm to 2/3 person
i have not created any server
and i have joined 3/4 server gssoc,gamezon,duhacks

big issue is i have tried to make new accout but it also got disabled just after making it and recived mail on them email id also.

please someone help me how can i reach GSSOC announcements

**Comment by shreya-paul-17** [comment, marked as answer]
Hey @Yashgabani845 ,
It sounds like your Discord account may have been mistakenly flagged. Here are some steps you can take to resolve this issue:

1. Appeal to Discord:
Go to the Discord Support [page](https://support.discord.com/hc/en-us/requests/new).
Select "Trust & Safety" as the category.
Explain your situation clearly, including that you believe your account was disabled in error and detail the activities you mentioned (e.g., only a few messages sent, no server creation, and limited server joining).

2. Avoid Creating New Accounts:
Since creating new accounts is also leading to them being disabled, it's best to stop trying this approach. It might be causing further issues with Discord's system.

3. Check Email for Details:
Carefully read any emails from Discord about the account disabling. Sometimes they include specific reasons or further instructions on how to appeal.

**Comment by MuraliDharan7** [comment]
Here are some steps you can take:

Fight the ban: There's a form to appeal the ban [appeal to discord ON support.discord.com]. Explain what happened. Tell them you barely DMed anyone and only joined a few servers (GSSoC, gamezon, duhacks). Be polite but firm.
GSSoC updates: While you wait, check the official GSSoC website [summer of code ON Google summerofcode.withgoogle.com] for announcements. They might also be on social media (Twitter, Facebook) – follow them for updates.
Discord on thin ice (if urgent): Need info ASAP? Make a new temporary Discord account but use it sparingly. Join the GSSoC server only and avoid too many DMs. Getting flagged again is not cool.
